src/Components/ExercisePage/Queries.tsx

```diff
@@ -2,13 +2,29 @@ import { ScrollableList } from '../Shared/ScrollableList';
 import { QueryListElement } from './QueryListElement';
 export const Queries = () => {
   const queryList = [
-    <QueryListElement query="b" />,
-    <QueryListElement query="c" />,
+    <QueryListElement query="A<> B.deadlock -> C.deadlock" result={true} />,
+    <QueryListElement query="A[] B.final -> C.final_state" result={true} />,
+    <QueryListElement query="A<> B.deadlock -> C.deadlock" result={false} />,
+    <QueryListElement query="A[] B.final -> C.final_state" result={true} />,
+    <QueryListElement query="A<> B.deadlock -> C.deadlock" result={true} />,
+    <QueryListElement query="A[] B.final -> C.final_state" result={true} />,
+    <QueryListElement query="A<> B.deadlock -> C.deadlock" result={true} />,
+    <QueryListElement query="A[] B.final -> C.final_state" result={true} />,
+    <QueryListElement query="A<> B.deadlock -> C.deadlock" result={true} />,
+    <QueryListElement query="A[] B.final -> C.final_state" result={true} />,
+    <QueryListElement query="A<> B.deadlock -> C.deadlock" result={true} />,
+    <QueryListElement query="A[] B.final -> C.final_state" result={true} />,
+    <QueryListElement query="A<> B.deadlock -> C.deadlock" result={true} />,
+    <QueryListElement query="A[] B.final -> C.final_state" result={true} />,
   ];
   return (
-    <>
-      <h1>Queries</h1>
+    <div className="flex w-2/6 h-4/6 pr-6 pb-6 pl-6 flex-col">
```

Comment:
Why use "*className*" and not "*class*"?

Reply:
It is the way React named the HTML attribute 'class'. It is very much the same thing.